boot/bootutil/src/loader.c

```diff
@@ -55,6 +55,10 @@
 #include <os/os_malloc.h>
 #endif
 
+#if defined(CONFIG_MCUBOOT_ACTION_HOOKS)
```

Comment:
This use of `mcuboot_status.h` also should no longer need ifdefs. Just include it, and call `mcuboot_status_change` below, like done in main.

Reply:
Fixed